modules/tensor_mechanics/tests/capped_mohr_coulomb/random1.i

```diff
@@ -0,0 +1,325 @@
+# Using CappedMohrCoulomb with tensile failure only
+# Plasticity models:
+# Tensile strength = 1MPa
+#
+# Lame lambda = 1GPa.  Lame mu = 1.3GPa
+#
+# A line of elements is perturbed randomly, and return to the yield surface at each quadpoint is checked
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 1000
+  ny = 1234
+  nz = 1
+  xmin = 0
+  xmax = 1000
+  ymin = 0
+  ymax = 1234
+  zmin = 0
+  zmax = 1
+[]
+
+
+[Variables]
+  [./disp_x]
+  [../]
+  [./disp_y]
+  [../]
+  [./disp_z]
+  [../]
+[]
+
+[Kernels]
```

Comment:
We should switch to using the [Modules/TensorMechanics/Master] Action in all models.  It you haven't looked at it yet, it's really nice -- it can set up all of the standard stress output variables, and sets up the kernels, variables (optionally), and strain calculator.